bids/layout/layout.py

```diff
@@ -107,7 +107,8 @@ def __init__(self, root, validate=True, index_associated=True,
                           "project root. Every valid BIDS dataset must have "
                           "this file.")
         else:
-            self.description = json.load(open(target, 'r'))
+            with open(target, 'r') as desc_fd:
```

Comment:
According to #70/#71, it looks like all of these should also have `encoding='utf-8'` options in `open`.